### PR TITLE
feat: substitute `__NOW__` to current timestamp in nanoseconds

### PR DESCRIPTION
**Pull request by BugenZhao**

### CODE REVIEW

**Review by stdrc — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Unreleased
 
 * runner: `RecordOutput` is now returned by `Runner::run` (or `Runner::run_async`). This allows users to access the output of each record, or check whether the record is skipped.
+* substitution: add a special variable `__NOW__` which will be replaced with the current timestamp in nanoseconds.
```
Comment (xxchan) [inline CHANGELOG.md:11]:
```suggestion
* substitution: add a special variable `__NOW__` which will be replaced with the current UNIX timestamp in nanoseconds.
```

**Review by xxchan — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (xxchan) [inline CHANGELOG.md:11]:
```suggestion
**Breaking changes**:
* runner: `RecordOutput` is now returned by `Runner::run` (or `Runner::run_async`). This allows users to access the output of each record, or check whether the record is skipped.
* substitution: add a special variable `__NOW__` which will be replaced with the current Unix timestamp in nanoseconds.
```

**Comment by stdrc** [comment]
When to merge?

**Comment by xxchan** [comment]
@stdrc FYI new version is released, and also upgraded in RisingWave

**Pull request merged**